templates/addthis-wrapper.html.twig

```diff
@@ -3,10 +3,10 @@
  * @file
  * Default theme implementation to display a node.
  * Available variables:
- * - element: AddThis Wrapper to be rendered
- * - element['']: Something else to render.
+ * - element: original AddThis wrapper object to reference
+ * - element['#markup']: Html markup to render for AddThis output
  *
  */
 #}
 
-{{ element.addthis_wrapper }}
\ No newline at end of file
+{{ element['#markup'] }}
```

Comment:
Is there a difference between `element['#markup']` and `element`? Normally there wouldn't be. If there is, my guess is that it's because we added the `#markup` item in the middle of the render process after it would have checked for it.

As with the other comment, I think we should move all that logic to here instead of the #pre_render function.